### ISSUE TITLE
Cannot assign to read only property 'defaultVal' - Chrome 38

### ISSUE THREAD

**Issue by manrueda** [freqdec/datePicker#47]
Hi @freqdec,

With the last version of the chrome, version 38, the script throws this error when initialize a DatePicker.

```
Uncaught TypeError: Cannot assign to read only property 'defaultVal' of %d/%m/%Y
```

Greatings-

**Comment by manrueda** [comment]
Sorry @freqdec , i had an old version of the code. The master code works fine...

**Comment by manrueda** [comment]
The minified version is not updated... still have this issue...

**Comment by Xaerxess** [comment]
It's probably fixed by #43. Can you check code from @dkeeghan's fork?

**Comment by freqdec** [comment]
Fixed in latest version guys.
